assets/utils/h5p.js

```diff
@@ -989,11 +990,21 @@ function itemUntranslatable(property, value, parent) {
       return false;
       break;
     case 'default':
-
+      if (typeof value !== 'string') {
+        return true;
+      }
       if (parent.type === 'select' || parent.widget === 'colorSelector') {
         return true;
       }
-
+      if (new RegExp(/^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/).test(value) === true || ['rgb(', 'hsv '].indexOf(value.substr(0, 4)) !== -1) { // color codes
+        return true;
+      }
+      if (new RegExp(/<\/?[a-z][^>]*>/ig).test(value) === true) { // html tags
```

Comment:
I'm not really sure about the extent to which we're using html tags here, but I think this regex might not be sufficient, as it just checks if the string contains html-like tags. It will return true if it contains both html tags and text, and there might be cases where we still want to translate the text.

I believe that it should be fairly straightforward to just check if the text is empty after stripping html tags, as Oliver previously suggested.

Reply:
@devland I was not thinking of "black", "magenta" or whatever. No sane developer should use that in production code. I was thinking of "rgb()" etc.